Cargo.toml

```diff
@@ -12,11 +12,14 @@ edition = "2018"
 
 [features]
 default = ["geo-types"]
+geo-traits = ["dep:geo-traits", "dep:bytemuck"]
```

Comment:
How do you feel about making this a default dependency? I wouldn't imagine it would add too much to compilation time?

Reply:
Fine by me! As long as we keep it optional